CHANGELOG.md

```diff
@@ -7,10 +7,12 @@ This project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html
 ## Unreleased
 
 ### Breaking changes
+- Drop support for Rails < 6.1
```

Comment:
I'd append "This also drops support for MySQL, which had only been present for Rails < 6."